inc/features/blocks/experience/register.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Experience Block Server Side.
+ *
+ * @phpcs:ignoreFile HM.Files.FunctionFileName.WrongFile
+ *
+ * @package altis-experiments
+ */
+
+namespace Altis\Experiments\Features\Blocks\Experience;
+
+use const Altis\Experiments\ROOT_DIR;
+use function Altis\Experiments\Features\Blocks\get_block_settings;
+use function Altis\Experiments\Utils\get_asset_url;
+
+const BLOCK = 'experience';
+
+function setup() {
+	$block_data = get_block_settings( BLOCK );
+
+	// Queue up JS files.
+	add_action( 'enqueue_block_editor_assets', __NAMESPACE__ . '\\enqueue_assets' );
+
+	// Register the block.
+	register_block_type( $block_data['name'], [
+		'attributes' => $block_data['settings']['attributes'],
+		'render_callback' => __NAMESPACE__ . '\\render_block',
+	] );
+}
+
+function enqueue_assets() {
+	wp_enqueue_script(
+		'altis-experiments-features-blocks-experience',
+		get_asset_url( 'features/blocks/experience.js' ),
+		[
+			'wp-plugins',
+			'wp-blocks',
+			'wp-i18n',
+			'wp-editor',
+			'wp-components',
+			'wp-core-data',
+			'wp-edit-post',
+			'altis-experiments-features-blocks-variant',
+			// Exports the audience UI modal component.
+			'altis-analytics-audience-ui',
+		],
+		null
+	);
+
+	wp_add_inline_script(
+		'altis-experiments-features-blocks-experience',
+		sprintf(
+			'window.Altis = window.Altis || {};' .
+			'window.Altis.Analytics = window.Altis.Analytics || {};' .
+			'window.Altis.Analytics.Experiments = window.Altis.Analytics.Experiments || {};' .
+			'window.Altis.Analytics.Experiments.BuildURL = %s;',
+			wp_json_encode( plugins_url( 'build', dirname( __FILE__, 4 ) ) )
+		),
+		'before'
+	);
+
+	// Queue up editor CSS.
+	wp_enqueue_style(
+		'altis-experiments-features-blocks-experience',
+		plugins_url( 'inc/features/blocks/experience/edit.css', ROOT_DIR . '/plugin.php' ),
+		[],
+		'2020-05-18-02'
```

Comment:
Should this be a constant so we can bump easier? Also JS should use the same ver I think.

Reply:
JS uses a content hash in the file name so version isn't needed. Think this is the only CSS file so I'd rather hold off of any updates here.